get-pmm.sh

```diff
@@ -223,19 +223,25 @@ run_docker() {
 # If any PMM server instance is run - stop and backup it.
 #######################################
 start_pmm() {
-  msg "Starting PMM server..."
+  msg "Fetching docker image...\n"
   run_docker "pull $repo:$tag 1> /dev/null"
 
+  msg "Checking data volume...\n"
+  pmm_archive="$(date "+%F-%H%M%S")"
   if ! run_docker "inspect pmm-data 1> /dev/null 2> /dev/null"; then
     run_docker "create -v /srv/ --name pmm-data $repo:$tag /bin/true 1> /dev/null"
-    msg "Created PMM Data Volume: pmm-data"
+    msg "Created PMM Data Volume: pmm-data\n"
+  else 
+    msg "\n\t${YELLOW}Existing PMM Data volume found, renaming to pmm-data-$pmm_archive${NOFORMAT}\n"
+    run_docker "rename pmm-data pmm-data-$pmm_archive"
```

Comment:
I checked this command and looks like it doesn't work for volumes. couldn't find anything in documentation about renamin data volumes. issue in moby project still open https://github.com/moby/moby/issues/31154

Reply:
what error are you getting?  I ran this on ubuntu and it successfully renames the volume and the new container references the new volume (the old container would also reference the new volume since they're both looking for pmm-data from the initial docker run).  I'll check on mac shortly but it seems to be working everywhere I'm trying it.  May be moot point if we agree that a proper backup is a smarter way to do upgrades.

Reply:
appears to work both on mac and ubuntu?

Reply:
tested on my mac
```
nurlan@Nurlans-MacBook-Pro action-slack-notify % docker volume create pmm-data
pmm-data
nurlan@Nurlans-MacBook-Pro action-slack-notify % docker rename pmm-data pmm-data-backup
Error response from daemon: No such container: pmm-data
Error: failed to rename container named pmm-data
nurlan@Nurlans-MacBook-Pro action-slack-notify % 
```